tokenserver/assignment/sqlnode/sql.py

```diff
@@ -192,6 +203,7 @@ def __init__(self, sqluri, create_tables=False, pool_size=100,
                  spanner_node_id=None, migrate_new_user_percentage=0,
                  **kw):
         self._cached_service_ids = {}
+        self._cache_migration_ttl = 0
```

Comment:
naming nit: as a reader, the name of this property doesn't help me understand how it's going to be used. Does it control a "cache migration" feature that I haven't encountered yet? Is it the number of seconds for which the migration should live? I suggest something like `_cached_migration_exp_at`, but I don't feel strongly enough about it to bikeshed in any detail 😀